docs/pip.md

```diff
@@ -0,0 +1,163 @@
+# Pip
+
+This document describes some of the more intricate details of Cachito support for pip. For a high
+level overview, look [here in the README](../README.md#pip).
+
+Cachito has a number of specific requirements when it comes pip packages. Some of those stem from
+the general ideas behind Cachito (e.g. reproducibility), some from the technical challenges of
+supporting a packaging system which defines most metadata through a Python executable. Read on for
+more details.
+
+## `requirements.txt`
```

Comment:
(optional): I do understand the reason behind having quotes here to make them codable but when you look at the README, it just looks bit off appearance wise. I'd rather just have it as regular title or put them in normal non-codable quotes.